added python3 version of program

jlefkoff opened the pull request.

jlefkoff committed:
added python3 version of program

gsalaman commented:
Question:  is the only non-backwards compatible thing in here the "Tkinter" (upper case T for python2) vs "tkinter" (lower case t for python3)?

jlefkoff committed:
made file cross compatible

jlefkoff commented:
@gsalaman was looking at my github profile and realized I never addressed your comment! Made the file work with both python 2 and 3. Hope you're doing well!

gsalaman commented:
Nice!!!!  Yeah, still doing some of the dawson stuff....

Hope college is going well!!!  Whatcha doing over the summer?
